Apps/AdminWebClient/src/ClientApp/src/services/restEmailAdminService.ts

```diff
@@ -30,13 +30,10 @@ export class RestEmailAdminService implements IEmailAdminService {
 
   public resendEmails(emailIds: string[]): Promise<string[]> {
     return new Promise((resolve, reject) => {
-      let headers: Dictionary<string> = {};
-      headers["Content-Type"] = "application/json; charset=utf-8";
```

Comment:
Without this it was having issues reaching the server with the correct format. Did you test that it does not break it?